specs/ldm/link-discovery-management-spec.html

```diff
@@ -303,32 +316,39 @@ <h3>Terminology</h3>
         <dl class="spaced">
           <dt><dfn>Link</dfn></dt>
           <dd>
-            An assertion or RDF triple consisting of a (subject, predicate, object) that manifests an instance of a relationship between the referenced subject and object.
+            An assertion or RDF triple consisting of a (subject, predicate, object) that manifests an instance of a
+            relationship between the referenced subject and object.
           </dd>
 
           <dt>LDM Client</dt>
           <dd>
-            An implementation of an application that consumes services provided by LDM servers to discover the scope of available links, and incoming links to a particular set of target resources.
+            An implementation of an application that consumes services provided by LDM servers to discover the scope of
+            available links, and incoming links to a particular set of target resources.
           </dd>
 
           <dt>LDM Server</dt>
           <dd>
-            A server implementing the OSLC Link Discovery Management specification. OSLC LDM clients consume
-            services provided by LDM Servers to discover incoming links. The use of the terms Client and Server are intended to distinguish typical
-            consumers and providers of OSLC resources in a distributed environment based on REST. A particular
-            application component could be a client for some OSLC domain services and a server for the same or another
-            domain.
+            A server implementing the OSLC Link Discovery Management specification. OSLC LDM clients consume services
+            provided by LDM Servers to discover incoming links. The use of the terms Client and Server are intended to
+            distinguish typical consumers and providers of OSLC resources in a distributed environment based on REST. A
+            particular application component could be a client for some OSLC domain services and a server for the same
+            or another domain.
           </dd>
           <dt>Concept Resource</dt>
```

Comment:
Why is this spec redefining a definition from another spec?

Reply:
done - referred to OSLC-Server